install/Knative-with-Minishift.md

```diff
@@ -0,0 +1,256 @@
+# Knative Install on OpenShift
+
+This guide walks you through the installation of the latest version of [Knative
+Serving](https://github.com/knative/serving) on an
+[OpenShift](https://github.com/openshift/origin) using pre-built images and
+demonstrates creating and deploying an image of a sample "hello world" app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+These instructions will run an OpenShift 3.11 (Kubernetes 1.11) cluster on your
+local machine using [minishift](https://docs.okd.io/latest/minishift/getting-started/index.html)
+to test-drive Knative.
+
+## Minishift setup
+
+- [Prepare virtualization environment](https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html) to be used with Minishift
+- Download the archive for your operating system from the [Minishift v1.25.0](https://github.com/minishift/minishift/releases/tag/v1.25.0) Release page and extract its contents
+- Copy the `minishift` binary from extracted directory to your preferred location like /usr/local/bin
+- Ensure `minishift` is in your PATH by running the command:
+
+```shell
+# returns minishift v1.25.0+90fb23e
+minishift version
+```
+
+## Configure and start minishift
+
+The following details the bare minimum configuration required to setup minishift for running Knative:
+
+```shell
+
+# make sure you have  a profile is set correctly, e.g. knative
+minishift profile set knative
+
+# Pinning to the right needed OpenShift version, anything >= v3.11.0
+minishift config set openshift-version v3.11.0
+
+# minimum memory required for the minishift VM
+minishift config set memory 8GB
+
+# the minimum required vCpus for the minishift vm
+minishift config set cpus 4
+
+# extra disk size for the vm, default is 20g
+minishift config set disk-size 50g
```

Comment:
With Minishift v1.25 , you do not need to set these explicitly, `minishift start --memory 8GB --cpus 4 --disk-size 50GB   --openshift-version latest ` would add these to persistent config.

Reply:
@kameshsampath s/50g/50GB/